scripts/qa_crowbarsetup.sh

```diff
@@ -4608,7 +4608,10 @@ function onadmin_prepare_crowbar_upgrade()
         if crowbarctl upgrade repocheck crowbar --format plain | grep "missing" ; then
             complain 11 "Some repository is missing on admin server. Cannot upgrade."
         fi
-        safely crowbarctl upgrade prepare
+        # FIXME: crowbarctl command can time out easily
```

Comment:
would be ok until we have the background job for the prepare. we just need to find out some sane timeout value here for the time being